lib/plugins/place_block.js

```diff
@@ -4,6 +4,8 @@ module.exports = inject
 
 function inject (bot) {
   async function placeBlockWithOptions (referenceBlock, faceVector, options) {
+    const {x, y, z} = referenceBlock.position
```

Comment:
Need to add the face vector to this, which is the ultimate position of the placed block. What we're trying to avoid is people passing blocks that are "buried" behind other blocks as the `referenceBlock`.

Reply:
would that be:
```js
referenceBlock.position.plus(faceVector)
```
?